### ISSUE TITLE
Hello

### ISSUE THREAD

**Issue by BuddhaWay** [DamienDoury/Mobamon#2]
Hey are you still working on this project?

**Comment by DamienDoury** [comment]
This project had been shut down due to legal issues.

**Comment by BuddhaWay** [comment]
o shit really? Damn Nintendo

**Comment by DamienDoury** [comment]
Well, that's their Intellectual Property. As a video game developer myself, I would react the same. That's totally understandable. 
That's life _shrug_

**Comment by BuddhaWay** [comment]
Yeah I understand but its stopping people from making something awesome

**Comment by BuddhaWay** [comment]
Sorry I have been on a free software trip and I dont know why it has not translated to gaming

**Comment by BuddhaWay** [comment]
I dont know maybe my mind is clouded by watching open source documentaries lately
